pkg/sql/pgwire/testdata/pgtest/portals

```diff
@@ -0,0 +1,29 @@
+# Verify that a completed portal can't be re-executed.
```

Comment:
have you tried this inside a transaction? I think maybe I've started destroying portals in between transactions at some point, so now you get an error that the portal doesn't exist, but within a transaction the portals are still as stateless as they've always been?